### PR TITLE
Update EmpathySimulationHandler.cs

### PR DESCRIPTION
**Pull request by JohnHolmesII**
Adds a little spice, since people are starting to catch on. This has not been tested, just fyi.

### CODE REVIEW

File: CompatBot/EventHandlers/EmpathySimulationHandler.cs [hunk 1]
```diff
@@ -42,7 +42,8 @@ public static async Task OnMessageCreated(MessageCreateEventArgs args)
                 return;
 
             var similarList = queue.Where(msg => content.Equals(msg.Content, StringComparison.InvariantCultureIgnoreCase)).ToList();
-            if (similarList.Count > 2 && similarList.Select(msg => msg.Author.Id).Distinct().Count() > 2)
+            var rng = new Random();
+            if (similarList.Count > rng.Next(2, 5) && similarList.Select(msg => msg.Author.Id).Distinct().Count() > 2)
```
Comment (13xforever) [inline CompatBot/EventHandlers/EmpathySimulationHandler.cs]: you probably meant to check if unique author count is random, not that there were random number of messages with the same content; also if you create new random every time, just inline it

**Comment by JohnHolmesII** [comment]
You're comment actually got me thinking. Any plans for monitoring if a *single* user keeps posting the same thing over and over? Like a certain someone with a penchant for roosters? Not entirely related to this pr, but the principle and code would wind up similar.

**Comment by 13xforever** [comment]
there's #416

**Comment by JohnHolmesII** [comment]
Not quite, I was hoping to have the bot bully those users. A lot.

**Pull request merged**